test/integration/read-property-multiple.spec.js

```diff
@@ -25,16 +25,16 @@ describe('bacstack - readPropertyMultiple integration', function() {
     client.readPropertyMultiple('127.0.0.1', requestArray, function(err, response) {
       var object = utils.propertyFormater(response.values[0].values);
       expect(response.values[0].objectIdentifier).to.deep.equal({type: 29, instance: 31});
-      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7}]);
+      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7, encoding: 0}]);
```

Comment:
When passing parameters in form of an object, implicit with defaults is the way to go in JS. Furthermore, it would better cover the real world, as people already using the lib without encoding parameterand would therefore minimize the impact.